rllib/models/action_dist_v2.py

```diff
@@ -0,0 +1,119 @@
+from typing import Tuple
+import gym
+import abc
+
+from ray.rllib.utils.annotations import ExperimentalAPI
+from ray.rllib.utils.typing import TensorType, Union, ModelConfigDict
+
+
+@ExperimentalAPI
+class ActionDistributionV2(abc.ABC):
```

Comment:
We should really take this opportunity and get rid of the restriction to actions. Can we just call this `Distribution`?
This would then cover models that predict next states, rewards, actions, etc.. as well as serve inside variational autoencoders.

Reply:
Yeah, I agree. fixed.